deployment/terraform/variables.tf

```diff
@@ -15,9 +15,9 @@ variable "SUBSCRIPTION_ID" {
 }
 
 variable "K8S_KUBE_CONFIG" {
-  description = "Path to Kube Config File"
+  description = "Path to Kube Config File (can be created through prepare environment)"
```

Comment:
"prepare environment" could be changed to "prepare_environment.sh" to make most clear what the name of the script is.